stunnel/stunnel.ml

```diff
@@ -126,6 +122,10 @@ let getpid ty =
   | Nopid ->
       failwith "No pid!"
 
+type verify = VerifyPeer | CheckHost
+
+type config = {sni: string option; verify: verify; cert_bundle_path: string}
```

Comment:
i think `verification_config` is a better name here (to avoid confusion with 'the stunnel config', which refers to the client config as a whole)

Reply:
Done